projects/moment/package.json

```
@@ -5,12 +5,12 @@
   "author": "PL <kuhnroyal@gmail.com>",
   "license": "MIT",
   "peerDependencies": {
-    "@angular/cdk": "^13.1.1",
-    "@angular/common": "^13.1.1",
-    "@angular/core": "^13.1.1",
-    "@angular/material": "~13.1.1",
-    "@angular/material-moment-adapter": "~13.1.1",
-    "@mat-datetimepicker/core": "^9.0.0"
+    "@angular/cdk": "^13.2.1",
+    "@angular/common": "^13.2.1",
+    "@angular/core": "^13.2.1",
+    "@angular/material": "~13.2.1",
+    "@angular/material-moment-adapter": "~13.2.1",
```

@eranboww many thanks, you're right that is incorrect! but, I think a better patch would only change lines 11 and 12 as following:

```suggestion
    "@angular/material": "^13.1.1",
    "@angular/material-moment-adapter": "^13.1.1",
```

that would align it with the [`@mat-datetimepicker/core`](https://github.com/kuhnroyal/mat-datetimepicker/blob/release/projects/core/package.json#L16). it will allow the usage of `@angular/material` up to v14.

a changeset should always be as minimal as possible. as long as there are no major / critical fixes above `@angular/core@13.1.1` I would not force people to upgrade (although it's best practice). the `^` still allows usage of newer versions, tho.